### PR TITLE
index.html: Add github icon and change footer address

### PR DESCRIPTION
**Pull request by sagar-kalra**
Add github icon in the menu bar and change Footer address

Fixes https://github.com/fossasia/fossasia.org/issues/168

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks! Please provide a screenshot and test page.

**Comment by sagar-kalra** [comment]
Here is the test link :- https://sagar-kalra.github.io/fossasia.org/
screenshot from 2017-09-18 11-45-01

**Comment by mariobehling** [comment]
Why is there a reference to a Jekyll theme? Don't add "FOSSASIA" twice. I am unclear what you are trying to do.

**Comment by sagar-kalra** [comment]
I ll correct it. Sorry, for the mistake.

**Comment by mariobehling** [comment]
Github icon not aligned and in the same size as other icons.

screenshot from 2017-09-18 11-57-39

**Comment by sagar-kalra** [comment]
i changed everything you asked. I also increased the size of github icon, but i am unsuccessful in positioning it as same as other icons.
This problem also persists in bottom social links display.
screenshot from 2017-09-18 22-28-53

File: index.html [hunk 1]
```diff
@@ -123,6 +123,7 @@
 									<li class="hidden-sm hidden-xs"><a href="http://blog.fossasia.org/contact/" target="_self">Contact</a></li>											
 									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
 									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
+<li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
```
Comment (dilpreetsio) [inline index.html]: Please add `class="social-link hidden-md hidden-sm hidden-xs"` to `li` here that will solve the icon alignment issue.

**Review by dilpreetsio — CHANGES_REQUESTED** [review]

**Comment by sagar-kalra** [comment]
https://sagar-kalra.github.io/fossasia.org/ 
screenshot from 2017-09-19 23-54-06

**Comment by dilpreetsio** [comment]
@sagar-kalra Looks good, can you please squash the commits?

**Comment by sagar-kalra** [comment]
i am trying squashing them into 1, but there is no success yet. Can you help me out? @geekyd

**Comment by isuruAb** [comment]
@sagar-kalra  follow these steps
git rebase -i HEAD~4
change all picks to squash  excerpt top one
press esc then :wq and enter
again esc then :wq
git push -f origin gh-pages

**Review by isuruAb — COMMENTED** [review]
@sagar-kalra follow these steps
follow these steps
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/#answer-5189600

**Comment by sagar-kalra** [comment]
when i do rebase it shows all commits i made in different issues also.

**Comment by isuruAb** [comment]
@sagar-kalra  as a good practise always start every new issue in new branch of master(in this case gh-pages)

**Comment by sagar-kalra** [comment]
yeah, i made a mistake this time. Is there any solution of this problem now?

**Comment by isuruAb** [comment]
you can reset your branch
rebase it from upstream/gh-pages and cherry-pick commits that you need for this PR.

**Comment by sagar-kalra** [comment]
https://sagar-kalra.github.io/fossasia.org/ 
Done.
screenshot from 2017-09-20 13-28-01

**Review by dilpreetsio — APPROVED** [review]

**Pull request merged**